internal/common_context.go

```diff
@@ -151,7 +158,13 @@ type QueryParamInvalidTypeError struct {
 }
 
 func (e QueryParamInvalidTypeError) Error() string {
-	return fmt.Errorf("param %s=%s is not of type %s: %w", e.ParamName, e.ParamValue, e.ExpectedType, e.Err).Error()
+	return fmt.Sprintf("%s: %s", e.DetailMsg(), e.Err.Error())
```

Comment:
This may panic if e.Err is nil

This might be better

```suggestion
	return fmt.Sprintf("%s: %s", e.DetailMsg(), e.Err)
```